### PR TITLE
Feature multi game play into host game

### PR DESCRIPTION
**Pull request by Leonard1020**
This will add the multiplayer gameplay to the connections that have already been made in host game

### CODE REVIEW

File: app/src/main/java/com/seniordesign/wolfpack/quizinator/Activities/HostGameActivity.java [hunk 1]
```diff
@@ -341,16 +344,30 @@ public void onClick(DialogInterface dialog, int id) {
         info.show();
     }
 
-    public void loadRuleInActivity(Rules rule) {
```
Comment (Leonard1020) [inline app/src/main/java/com/seniordesign/wolfpack/quizinator/Activities/HostGameActivity.java]: Removed this method and loadCardInActivity(...) and replaced it with startMultiplayerGamePlay(...)

File: app/src/main/java/com/seniordesign/wolfpack/quizinator/WifiDirect/ConnectionService.java [hunk 2]
```diff
@@ -381,11 +393,30 @@ private void onPushOutData(String data) {
     }
 
     /*
+     * @author leonardj (11/4/16)
+     */
+    public static boolean sendMessage(int code, String message) {
```
Comment (Leonard1020) [inline app/src/main/java/com/seniordesign/wolfpack/quizinator/WifiDirect/ConnectionService.java]: Static helper method to send string messages of type code. This has been really helpful.

File: app/src/main/java/com/seniordesign/wolfpack/quizinator/Activities/ManageGameplayActivity.java [hunk 3]
```diff
@@ -128,8 +139,30 @@ protected void onResume() {
...
     @Override
     protected void onPause() {
+        gameplayTimerRunning.cancel();
         super.onPause();
         deckDataSource.close();
         rulesDataSource.close();
     }
+
+    /*
+     * @author farrowc 10/14/2016
```
Comment (Jimmy9222) [inline app/src/main/java/com/seniordesign/wolfpack/quizinator/Activities/ManageGameplayActivity.java]: Not sure the date on this is correct...

**Comment by Jimmy9222** [comment]
Looks great, glad this got done on time

**Pull request merged**